hal/include/sonar.h

```diff
@@ -61,8 +63,10 @@ typedef struct
 	float 	min_distance; 		///< Minimum distance the sensor can measure in centimeters
 	float 	max_distance; 		///< Maximum distance the sensor can measure in centimeters
 	float 	current_distance;	///< Measured distance in meters
+	float	current_velocity;	///< Computed velocity in m/s
 	float 	covariance; 		///< Measurement covariance in centimeters, 0 for unknown / invalid readings
 	bool 	healthy;			///< Indicates whether the current measurement can be trusted
+	bool	healthy_vel;		///< Indicated whether the velocity estimation can be trusted
```

Comment:
@ndousse 
I don't see the point for adding the healthy_vel object, since it will be healthy if and only if the sensor is healthy, isn't it ?